invenio_theme/assets/semantic-ui/less/invenio_theme/theme/views/item.overrides

```diff
@@ -1,3 +1,7 @@
 /***********************************************
          Invenio Theme Item Overrides
 ***********************************************/
+
+.ui.items > .item .extra > *:last-child {
```

Comment:
could you add a comment why we need this change?

Reply:
To reduce a bit the space between the request Icon (the conversation one) and the rest of the content of the request in the request list view.

With css rule:
![Screenshot from 2022-09-06 13-37-38](https://user-images.githubusercontent.com/25476209/188629089-89031c4f-42b8-4738-b28c-4122b6f7eb28.png)

Without:
![Screenshot from 2022-09-06 13-37-44](https://user-images.githubusercontent.com/25476209/188629128-7cf9975b-d6f8-4efb-876e-562a67ecbb7b.png)

I agree that it's purpose can be quite confusing, so I changed it to use a class selector specific for this case, in order to make it easy to understand it's use by just reading the selector.